addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/discovery/RFXComBridgeDiscovery.java

```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.rfxcom.discovery;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+import jd2xx.JD2XX;
+
+import org.eclipse.smarthome.config.discovery.AbstractDiscoveryService;
+import org.eclipse.smarthome.config.discovery.DiscoveryResult;
+import org.eclipse.smarthome.config.discovery.DiscoveryResultBuilder;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.eclipse.smarthome.core.thing.ThingUID;
+import org.openhab.binding.rfxcom.RFXComBindingConstants;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link RFXComBridgeDiscovery} is responsible for discovering new RFXCOM
+ * transceivers.
+ * 
+ * @author Pauli Anttila - Initial contribution
+ * 
+ */
+public class RFXComBridgeDiscovery extends AbstractDiscoveryService {
+
+	private final static Logger logger = LoggerFactory
+			.getLogger(RFXComBridgeDiscovery.class);
+
+	public RFXComBridgeDiscovery() {
+		super(RFXComBindingConstants.SUPPORTED_BRIDGE_THING_TYPES_UIDS, 10);
+	}
+
+	@Override
+	public Set<ThingTypeUID> getSupportedThingTypes() {
+		return RFXComBindingConstants.SUPPORTED_BRIDGE_THING_TYPES_UIDS;
+	}
+
+	@Override
+	public void startScan() {
+		discoverRfxcom();
+	}
+	
+	/*
+	 * (non-Javadoc)
+	 * 
+	 * @see org.eclipse.smarthome.config.discovery.AbstractDiscoveryService#
+	 * startBackgroundDiscovery()
+	 */
+
+	@Override
+	protected void startBackgroundDiscovery() {
+		discoverRfxcom();
```

Comment:
Background discovery should permanently scan for devices - so it should be a job that is scheduled regularly or even better a callback that is called by some external mechanism (if the external system supports this)